src/client/containers/Modals/SetRatesTxModal/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react';
+import styled from 'styled-components';
+
+import { useAppTranslation } from '@hooks';
+import { ModalTx } from '@components/common';
+import { SetRatesTx } from '@components/app';
+
+const StyledSetRatesTxModal = styled(ModalTx)``;
+export interface SetRatesModalProps {
+  onClose: () => void;
+}
+
+export const SetRatesTxModal: FC<SetRatesModalProps> = ({ onClose, ...props }) => {
+  const { t } = useAppTranslation('common');
+
+  const onSelectedCreditLineChange = () => {
```

Comment:
on change handlers not needed

Reply:
will address this in a later PR for setRates. Lots of copy pasta from copying another component